### PR TITLE
OWL Gantt View (Community)

### PR DESCRIPTION
**Pull request by Polymorphe57**
for enterprise pull request see: odoo/enterprise#30816

### CODE REVIEW

File: addons/mrp/views/mrp_workorder_views.xml [hunk 1]
```diff
@@ -339,8 +339,8 @@
                         <div class="row g-0">
                             <div class="col">
                                 <ul class="ps-1 mb-0 list-unstyled">
-                                    <li><strong>Start Date: </strong> <t t-out="userTimezoneStartDate.format('L LTS')"/></li>
-                                    <li><strong>Stop Date: </strong> <t t-out="userTimezoneStopDate.format('L LTS')"/></li>
+                                    <li><strong>Start Date: </strong> <t t-out="date_planned_start.toFormat('F')"/></li>
```
[the remark this answers is not in the document]
Comment (brboi) [reply addons/mrp/views/mrp_workorder_views.xml:342]: ¯\_(ツ)_/¯

File: addons/mrp/models/mrp_workorder.py [hunk 2]
```diff
@@ -556,46 +556,39 @@ def _plan_workorder(self, replan=False):
     @api.model
     def gantt_unavailability(self, start_date, end_date, scale, group_bys=None, rows=None):
         """Get unavailabilities data to display in the Gantt view."""
-
```
Comment (Polymorphe57) [inline addons/mrp/models/mrp_workorder.py:559]: I think the feature did work "by chance": if workcenter_id is not found in the group_bys, the records in a given row can be associated with different workcenters and I don't see the point to mix the unvavailabilities in that case.

File: addons/web/static/src/webclient/actions/action_hook.js [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 import { onMounted, useComponent, useEffect, useExternalListener } from "@odoo/owl";
 
-const scrollSymbol = Symbol("scroll");
+export const scrollSymbol = Symbol("scroll");
```
Comment (brboi) [inline addons/web/static/src/webclient/actions/action_hook.js:5]: needed for the gantt renderer (virtual list initial scroll position)

File: addons/web/static/src/views/record.js [hunk 4]
```diff
@@ -9,8 +9,9 @@ const defaultActiveField = { attrs: {}, options: {}, domain: "[]", string: "" };
 
 class _Record extends Component {
     setup() {
+        const resId = this.props.info.resId || this.props.info.initialValues?.id;
```
Comment (Polymorphe57) [inline addons/web/static/src/views/record.js:12]: we will not use this commit anymore (if Avatar comp created as expected)

**Comment by brboi** [comment]
closing in favor of
https://github.com/odoo/odoo/pull/110819
https://github.com/odoo/enterprise/pull/36202

**Pull request closed without merging**